2_process/src/calculate_toha.R

```diff
@@ -98,6 +98,17 @@ interp_hypso_to_match_temp_profiles <- function(wtr, hypso) {
 #  of benthic areas between each depth
 benthic_areas <- function(depths, areas){
```

Comment:
Yes, I would suggest changing the argument name here -or- preprocessing the `H` vector to be `depth` in your pipeline.

Reply:
Is this what I would subtract out? https://github.com/USGS-R/lake-temperature-model-prep/blob/master/7_config_merge/src/munge_lake_info.R#L68

Reply:
I wouldn't assume that value will always be the crest height, as someday we should set it with actual elevation of the lake. 

You can calculate depth from the heights, noting that the surface elevation is there as well as the deepest?

Reply:
Thought all the TOHA might be off but forgot that we did this correction within the TOHA calc itself, so followed suit with my udpate https://github.com/USGS-R/lake-temperature-out/blob/master/2_process/src/calculate_toha.R#L9-L12

Reply:
good catch. So looks like we won't have to re-run all of those :+1: